### ISSUE TITLE
2.80 search crash when redo

### ISSUE THREAD

**Issue by Gvgeo** [Gvgeo/Pivot-Painter-for-Blender#2]
```
>	blender.exe!rna_ID_refine(PointerRNA * ptr) Line 406	C
 	blender.exe!RNA_id_pointer_create(ID * id, PointerRNA * r_ptr) Line 131	C
 	blender.exe!id_search_add(const bContext * C, TemplateID * template_ui, const int flag, const unsigned char * str, uiSearchItems * items, ID * id) Line 321	C
 	blender.exe!id_search_cb(const bContext * C, void * arg_template, const unsigned char * str, uiSearchItems * items) Line 363	C
 	blender.exe!ui_searchbox_update(bContext * C, ARegion * ar, uiBut * but, const bool reset) Line 365	C
 	blender.exe!ui_textedit_begin(bContext * C, uiBut * but, uiHandleButtonData * data) Line 3246	C
 	blender.exe!button_activate_state(bContext * C, uiBut * but, uiHandleButtonState state) Line 7349	C
 	blender.exe!ui_do_but_TEX(bContext * C, uiBlock * block, uiBut * but, uiHandleButtonData * data, const wmEvent * event) Line 4175	C
 	blender.exe!ui_do_button(bContext * C, uiBlock * block, uiBut * but, const wmEvent * event) Line 7060	C
 	blender.exe!ui_handle_button_over(bContext * C, const wmEvent * event, ARegion * ar) Line 7941	C
 	blender.exe!ui_handle_menu_button(bContext * C, const wmEvent * event, uiPopupBlockHandle * menu) Line 8903	C
 	blender.exe!ui_handle_menu_event(bContext * C, const wmEvent * event, uiPopupBlockHandle * menu, int level, const bool is_parent_inside, const bool is_parent_menu, const bool is_floating) Line 9469	C
 	blender.exe!ui_handle_menus_recursive(bContext * C, const wmEvent * event, uiPopupBlockHandle * menu, int level, const bool is_parent_inside, const bool is_parent_menu, const bool is_floating) Line 9991	C
 	blender.exe!ui_handler_region_menu(bContext * C, const wmEvent * event, void * UNUSED_userdata) Line 10156	C
 	blender.exe!wm_handler_ui_call(bContext * C, wmEventHandler_UI * handler, const wmEvent * event, int always_pass) Line 623	C
 	blender.exe!wm_handlers_do_intern(bContext * C, wmEvent * event, ListBase * handlers) Line 2699	C
 	blender.exe!wm_handlers_do(bContext * C, wmEvent * event, ListBase * handlers) Line 2945	C
 	blender.exe!wm_event_do_handlers(bContext * C) Line 3323	C
 	blender.exe!WM_main(bContext * C) Line 420	C
 	blender.exe!main(int argc, const unsigned char * * UNUSED_argv_c) Line 502	C
 	[External Code]	
```

**Comment by Gvgeo** [comment]
Extremely rare bug. nothing to do here.

**Comment by Gvgeo** [comment]
Most likely fixed with the final release of 2.80
